### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by SoCarbone** [johngodley/redirection#681]
```
Plugin: 3.0.1
WordPress: 4.9.2
PHP: 7.1.9
Browser: Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36 OPR/50.0.2762.67

Error: Unexpected token < in JSON at position 0
Action: redirection/v1/redirect POST
Params: {"id":0,"url":"https://agence-colombo.fr/nos-creations-digitales/toysrus-et-les-minions/","title":"","regex":false,"match_type":"url","action_type":"url","position":0,"group_id":1,"action_code":301,"action_data":{"url":"https://agence-colombo.fr/nos-creations-digitales/realisation-de-site-web-responsive/"}}
Code: 403 
Raw: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>403 Forbidden</title>
</head><body>
<h1>Forbidden</h1>
<p>You don't have permission to access /wp-json/redirection/v1/redirect
on this server.<br />
</p>
</body></html>

```

**Comment by johngodley** [comment]
Can you try using the 3.1 beta?

https://github.com/johngodley/redirection/releases/tag/3.1

I will be releasing this version tomorrow if you want to wait until then.

**Comment by SoCarbone** [comment]
I can wait the release.
Thank you for your reply.
Good evening

**Comment by johngodley** [comment]
I've now released 3.1. I will close this ticket for now, but please reopen if the problem persists
